frontend/src/features/maps/routes/MapCreateForm.tsx

```diff
@@ -127,6 +127,20 @@ export default function MapCreateForm() {
       privacy: mapInput.privacy,
       description: mapInput.description,
       location: !Number.isNaN(mapInput.location.latitude) ? mapInput.location : undefined,
+      // TODO: implement selector
```

Comment:
Please describe better what you mean here. Do you mean #494, then please link to the issue.

Reply:
I think this refers to #349 (#494 is a duplciate of that?).
Simply said when a map is created the user needs to specify the boundaries of the map. This boundary is then used by the heatmap (and maybe other layers like the base layer?).

Reply:
I made a distinction between these two issues (one for backend, other for frontend).

> Simply said when a map is created the user needs to specify the boundaries of the map. This boundary is then used by the heatmap (and maybe other layers like the base layer?).

Please read the use cases, the map needs to be created without the boundaries. So it must be possible to set a boundary afterwards.

Reply:
Should the map boundary be part of the base layer or part of the map? As far as I understand the following means it is part of map: `these boarders are stored in the map and not subjective to "alternatives"` (for the base_layer use case).
If so that would mean I simply need to set the field to nullable and add an option to update the boundary?

Reply:
Yes, the map boundary must be part of the map.